packages/x-charts/src/models/axis.ts

```diff
@@ -333,6 +333,12 @@ export type AxisConfig<
    * If `true`, Reverse the axis scaleBand.
    */
   reverse?: boolean;
+  /**
+   * Defines the axis scale domain based on the min/max values of series linked to it.
+   * - 'nice': Uses the `.nice()` method from d3-scale to round the domain at human friendly values.
+   * - 'strict': Set the domain to the min/max value displayed. No extras space is added.
+   */
+  domainLimit?: 'nice' | 'strict';
```

Comment:
Small change, not sure we need to mention d3 and nice

```suggestion
  /**
   * Defines the axis scale domain based on the min/max values of series linked to it.
   * - 'nice': Rounds the domain at human friendly values.
   * - 'strict': Set the domain to the min/max value provided. No extras space is added.
   */
  domainLimit?: 'nice' | 'strict';
```

Reply:
Updated